androidComposePrototype/src/androidTest/java/com/genesys/cloud/messenger/uitest/test/ComposePrototypeUITest.kt

```diff
@@ -248,4 +292,37 @@ class ComposePrototypeUITest : BaseTests() {
         }
         bye()
     }
+
+    @Test
+    fun testUnknownAgent() {
+        apiHelper.disconnectAllConversations()
+        enterDeploymentInfo(testConfig.humanizeDisableDeploymentId)
+        DefaultTokenStore("com.genesys.cloud.messenger").store(UUID.randomUUID().toString())
```

Comment:
Very minor suggestion. Since you're sending text before answering the conversation anyway, resetting the messenger token shouldn't be required. Keeping this is totally fine though if you prefer having a different token for every test for troubleshooting purposes.

Reply:
I see what you mean. I will take this out.